main.go

```diff
@@ -94,6 +105,22 @@ func main() {
 		c.Start()
 	}
 
+	exporter, err := prometheus.NewExporter(prometheus.Options{})
+	if err != nil {
+		glog.Fatalf("can't create prometheus exporter: %v", err)
+	}
+	view.RegisterExporter(exporter)
+
+	mux := http.NewServeMux()
+	mux.Handle("/metrics", exporter)
+	srv := &http.Server{
+		Addr:    *debugAddr,
+		Handler: mux,
+	}
+	go func() {
+		glog.Fatalf("cannot serve debug endpoint: %v", srv.ListenAndServe())
```

Comment:
Maybe this should be logged but not fatal? The debug endpoint isn't strictly necessary for Metacontroller to perform its main duties. If I'm relying on Metacontroller to manage my app, I wouldn't want it to give up just because, for example, the metrics endpoint couldn't bind due to a port conflict.

In any case, we should probably ignore `ErrServerClosed` here because we call `Shutdown()` below.

Reply:
You're right I remember fighting with this log line quite a bit I should have paid more attention. I'll change this to an `Errorf`